### DISCUSSION TITLE
Is there any relationship between cursor id and node id in Github v4 API?

### DISCUSSION THREAD

**Discussion by shivam05011996** [community/community#24825] [Apps, API and Webhooks]
<p>I am trying to do pagination for the issues in a repository, I want the next cursor id to be automatically calculated based on the node id, is there any possible way? Or is there a way to form the next cursor id based on the startCursor without accessing the nextCursor value?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>No, there isn’t a method to calculate the next cursor value outside of requesting it from the API.</p>
<p>Let us know if you have more questions.</p>
